string series
float64 xValue
float64 yValue